OPHD/UI/NotificationArea.cpp

```diff
@@ -130,7 +131,10 @@ void NotificationArea::onMouseMove(int x, int y, int /*dX*/, int /*dY*/)
 		{
 			mNotificationIndex = count;
 
-			mNotificationBriefRect = { 100, 100, 100, 25 };
+			const int stringWidth = mFont.width(mNotificationList[count].brief) + 8;
+			const int briefPositionX = positionX() - stringWidth - 4;
+
+			mNotificationBriefRect = { briefPositionX, rect.y, stringWidth, mFont.height() + 4 };

```

Comment:
Might be wise to use named constants for the magic numbers used here. Perhaps `constexpr int margin = 4;`. Then width can add `margin * 2`, while the others use `- margin` and `+ margin`.

At least, that's my rough interpretation of what those magic numbers represent. Maybe not since x and y margins might not be the same here.

Reply:
UIConstants has these already defined, managed to forget to change the magic numbers to the named constants. Will slate this for an update going forward.